ONSdigital/census-rm-docker-dev
# remove settings for pubsub exchange, queue & bindings

# Motivation and Context
The removed settings are set correctly in pubsub, don't need to be set here (they changed in the repo and ATs now failing)

# What has changed
removed env settings for pubsub queue, binding & exhange.

# How to test?
Use this docker dev and do a clean make up.  With master everything the ATs (on master) . should all pass

# Links
https://trello.com/c/HzhnnprB/1041-remove-pubsub-queue-settings-from-docker-dev
